app/src/main/java/org/oppia/android/app/drawer/ExitProfileDialogFragment.kt

```diff
@@ -17,22 +17,32 @@ class ExitProfileDialogFragment : DialogFragment() {
     // TODO(#1655): Re-restrict access to fields in tests post-Gradle.
     const val BOOL_IS_FROM_NAVIGATION_DRAWER_EXTRA_KEY =
       "BOOL_IS_FROM_NAVIGATION_DRAWER_EXTRA_KEY"
+    const val BOOL_IS_ADMINISTRATOR_CONTROLS_SELECTED_KEY =
+      "BOOL_IS_ADMINISTRATOR_CONTROLS_SELECTED_KEY"
+    const val INT_LAST_CHECKED_ITEM_KEY =
+      "INT_LAST_CHECKED_ITEM_KEY"
```

Comment:
Nit: move to a single line if it fits (per 100 character limit).

Reply:
Done